components/features/Signup/InitialSignup/SignupChoice/SignupChoice.tsx

```diff
@@ -33,7 +33,7 @@ const SignupChoice = ({
         variants={signupChoiceVariants}
         initial={isSelected ? "selected" : "unselected"}
         animate={isSelected ? "selected" : "unselected"}
-        className={styles["signup-choice"]}
+        className={`${styles["signup-choice"]} ${styles["button"]}`}
```

Comment:
Hello, Alex.
Correct me if I'm wrong. Is it true that now the class name depends not only on the sign up choice but also by using the button styles such that the classname now compose of two variables rather than 1?

Reply:
Hello, Jo.
Yes, in this case signup-choice and button classes will be implemented as styling on this component. I decide to create this class as to not clash with Rashad's button styling.

Reply:
Noted, Alex. Thanks for your clarification.